McBopomofoTests/ServiceProviderTests.swift

```diff
@@ -35,7 +35,7 @@ final class ServiceProviderTests: XCTestCase {
     func testExtractReading1() {
         let provider = ServiceProvider()
         let output = provider.extractReading(from: "消波塊")
-        XCTAssert(output == "ㄒㄧㄠ-ㄆㄛ-ㄎㄨㄞˋ")
```

Comment:
Perhaps we need some tests for polyphonic characters like this?

Reply:
> Perhaps we need some tests for polyphonic characters like this?

I think so.

And when a test fail, it's a perfect timing to realize whether it's a regression. When it's not a regression, it indicates a change of behavior, and a change of behavior better has a good reason and a consensus, IMO.

BTW, a change of behavior can also break backward compatibility by definition, and then it will need a rather significant release in the sense of SemVer.

+CC @lukhnos @mjhsieh @zonble

Reply:
Good idea, updated.